Panic in the block_fetcher

````
INFO [07-21|10:36:26.771] Imported new chain segment               blocks=1    txs=165    elapsed=2.517ms     numbe
r=10502239 hash="b850c9…c40562"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xe5c6c5]
goroutine 77 [running]:
github.com/ledgerwatch/turbo-geth/core/types.(*Block).NumberU64(...)
        /home/igor/turbo-geth/core/types/block.go:331
github.com/ledgerwatch/turbo-geth/eth.initPm.func3(0xc0005ea848, 0x1, 0x1, 0x2224a01, 0xc072ef8090, 0x6)
        /home/igor/turbo-geth/eth/handler.go:196 +0x65
github.com/ledgerwatch/turbo-geth/eth/fetcher.(*BlockFetcher).insert(0xc0003f8b60, 0xc01e75a5f0, 0x10, 0xc0725c0090
)
        /home/igor/turbo-geth/eth/fetcher/block_fetcher.go:711 +0x7ce
github.com/ledgerwatch/turbo-geth/eth/fetcher.(*BlockFetcher).loop(0xc0003f8b60)
        /home/igor/turbo-geth/eth/fetcher/block_fetcher.go:344 +0x47b
created by github.com/ledgerwatch/turbo-geth/eth/fetcher.(*BlockFetcher).Start
        /home/igor/turbo-geth/eth/fetcher/block_fetcher.go:202 +0x3f
exit status 2
````

Proposed fix is to to make `inserter` a no-op function in `eth/handler.go`, line 189

--- a/eth/handler.go
+++ b/eth/handler.go
@@ -187,6 +187,12 @@ func initPm(manager *ProtocolManager, engine consensus.Engine, chainConfig *para
 		return *headNumber
 	}
 	inserter := func(blocks types.Blocks) (int, error) {
+		// FIXME: Workaround for staged sync, just do no-op
+		if manager.blockchain == nil || manager.blockchain.CurrentBlock() == nil {
+			log.Info("Workaround for staged sync: IIIAAAIII")
+			return 0, err
+		}
+
 		// If sync hasn't reached the checkpoint yet, deny importing weird blocks.
 		//
 		// Ideally we would also compare the head block's timestamp and similarly reject

